### PR TITLE
Added const for user level

### PR DESCRIPTION
**Pull request by phecho**

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -24,6 +24,8 @@ class User extends Model implements AuthenticatableContract, CanResetPasswordCon
 {
     use Authenticatable, CanResetPassword, ValidatingTrait;
 
+    const LEVEL_ADMIN = 1;
+    const LEVEL_USER = 2;
```
Comment (GrahamCampbell) [inline app/Models/User.php]: Please add phpdoc, and a newline after this, and it looks good.

**Comment by GrahamCampbell** [comment]
A bit like in here:

image

**Comment by jbrooksuk** [comment]
:gift:

**Comment by GrahamCampbell** [comment]
Thank you very much. ;)

**Pull request merged**